src/pages/groupComparison/comparisonGroupManager/ComparisonGroupManager.tsx

```diff
@@ -130,6 +152,8 @@ export default class ComparisonGroupManager extends React.Component<IComparisonG
                                     markedForDeletion={this.props.store.isComparisonGroupMarkedForDeletion(group.uid)}
                                     restore={this.restoreGroup}
                                     rename={this.renameGroup}
```

Comment:
this is what I'm talking about in my comment https://github.com/alisman/cbioportal-frontend/pull/20/files#r270983948

Notice how for `rename` we can just pass a single function, this isn't recreated on each render.